index.html

```diff
@@ -12,36 +12,35 @@
 
 <body>
 
-
     <main>
-        <header class="Top-nav">
-            <a><img class="mylogo" src="image/My Logo.png" alt="MyLogo"></a>
-            </a>
-            <a class="icon-btn" id="hamburger"><img id=" menu-icon " src="image/Icon - Menu.svg "></a>
-            <nav class="mobile-menu" id="mobile-menu">
-                <button class="close-btn" id="close-menu"><img id=" menu-icon " src="image/cancel.png"></button>
-                <ul class="mobile-menu">
-                    <li><a class="item" href="#section1">Portfolio</a></li>
-                    <li>
-                        <a class="item" href="#section2">About</a>
-                    </li>
-                    <li>
-                        <a class="item" href="#section3">Contact</a>
-                    </li>
-                    <li>
-                        <a class="item" href="#section3"><img src="image/Email-icon.svg " alt="e-mail "></a>
+        <header>
+            <li>
+                <img class="mylogo" src="image/My Logo.png"></li>
+            <nav>
+                <ul class="menu">
+                    <li><a class="menuItem" href="#Home">Home</a></li>
+                    <li><a class="menuItem" href="#Profile">Profile</a></li>
+                    <li><a class="menuItem" href="#About">About</a></li>
+                    <li class="email-icon">
+                        <a class="menuItem" href="#Contacts">
+                            <img src="image/Email-icon.svg"></a>
```

Comment:
- [ ] Great job implementing required changes by the previous reviewer. However, there're few things you need to work on before this milestone could be approved.
  -  It is required to stick to the Figma design as close as possible. 
     The words on your popup menu are the tally with what we have 
     on the Figma design. Kindly follow the Figma design to the 
     letter.
  -  Kindly make your popup menu background to  cover the view point(the whole page.)

### Your Design
![your-menu](https://github.com/richie1988/website-pages/assets/26597432/8520344d-583a-41df-ac8d-ff85fa69eda8)

### Figma Design
![tem-menu](https://github.com/richie1988/website-pages/assets/26597432/d23274d7-5a70-4e0b-b717-83d7ba0f3984)